adding resources

Ateret2 opened the pull request:
https://github.com/NNLP-IL/Resources/commit/a3f5d8dc36e110dcf5f619a3c9ad583fa692fc67

Ateret2 committed:
try

Ateret2 committed:
try

Ateret2 committed:
adding 'Stop words'

Ateret2 committed:
Update README.rst

Ateret2 committed:
TED

Transcripts

Hilla-Merhav commented:
@Ateret2

Thank you for adding resources to the list! I have a couple of requests regarding the changes you made:
1. Could you please add a hyperlink (https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) to the stop word’s license {GPL 2}? 
2. I think OpenSubtitles might violate copyrights. Could you please remove this item from the list?

Ateret2 committed:
Merge pull request #1 from Ateret2/Ateret2-patch-2

TED

Ateret2 committed:
updates

deletion OpenSubtitles,
additing Sentiment Lexicon and Israeli Pop,
correcting licenses (ted, stop words)

Ateret2 committed:
Update README.rst

Ateret2 committed:
Update README.rst

Ateret2 committed:
adding old newspapers

Ateret2 committed:
Update README.rst

Ateret2 committed:
Update README.rst

Hilla-Merhav commented:
@Ateret2 

Thank you for amending the list and adding resources. I have a few more requests regarding the changes you made:
1. I couldn’t find a dataset to corpus in All-Rights (כל-זכות)’s Github link. Do you know the specific repository that includes the data?
2. Old Newspapers can be set under the category of Unannotated Corpora.
3. As far as I understand, Sentiment Lexicon’s license is GNU General Public License.
4. This branch has conflicts (probably due to some recent pull requests that have been merged). Could you fix it?
Thanks a lot!

Ateret2 committed:
Merge branch 'master' into master

Ateret2 committed:
Update README.rst

Ateret2 committed:
Update README.rst

Ateret2 committed:
sentiment lexicon- update license

Ateret2 committed:
Update README.rst
